app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="TextAppearance">
+        <item name="android:fontFamily">@font/bai_jamjuree_medium</item>
+        <item name="android:textColor">@color/pastel_purple</item>
+        <item name="android:textSize">16sp</item>
+    </style>
+
+    <style name="Heading1" parent="TextAppearance">
+        <item name="fontFamily">@font/bai_jamjuree_bold</item>
+        <item name="android:textSize">32sp</item>
```

Comment:
Extract these values to `dimens`. Personally I prefer naming them like `h1_text_size`, `body1_...` `info_..`
If you have different sizes for tablets, then you'll only need to override these values in the dimens file for tabled configuraitons